.github/workflows/pr-workflow.yml

```
@@ -48,3 +50,40 @@ jobs:
         run: |
           npm i -g tailwindcss
           make build
+
+  main-api:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)